### ISSUE TITLE
I can't play any missions

### ISSUE THREAD

**Issue by majinisinpu** [acemod/ACE3#5060]
**Arma 3 Version:** 64bit `1.68` (stable)
**CBA Version:** `3.2.1` (stable )
**ACE3 Version:** `3.9.1` (stable )

**Mods:**
I'm using this correction
http://withsix.com/p/arma-3/collections/hw_r40PGyUCaKcKDPwsaLQ/honhonhon-best

**Description:**
This message appears and the mission ends forcedly
![20170406134036_1](https://cloud.githubusercontent.com/assets/18363105/24738370/353552ec-1ad0-11e7-8606-f66043b24e34.jpg)

**Where did the issue occur?**
Singleplayer - as soon as mission has started

**RPT log file:**

[Arma3_x64_2017-04-05_23-32-13.zip](https://github.com/acemod/ACE3/files/901490/Arma3_x64_2017-04-05_23-32-13.zip)

**Comment by commy2** [comment]
Disable Battleye.

**Comment by majinisinpu** [comment]
I did it but the issue hasn't solved.

**Comment by commy2** [comment]
You also need the .dll files inside the `@ACE folder`. Dunno what else to tell you.

**Comment by majinisinpu** [comment]
This is my @ace3 folder.
![img_1148](https://cloud.githubusercontent.com/assets/18363105/24789261/1e5bb272-1bad-11e7-83f3-78020799b339.JPG)

**Comment by PabstMirror** [comment]
You might want to try using 32 bit exe.
Also, if you are using the launcher, double check that battleye is disabled.

**Comment by majinisinpu** [comment]
I tried 32bit before but the issue hasn't solved.

[Arma3_2017-04-07_14-06-31.zip](https://github.com/acemod/ACE3/files/905107/Arma3_2017-04-07_14-06-31.zip)

**Comment by PabstMirror** [comment]
Something is blocking these dlls from loading.

It's either battleye (if running and it can't self-update for some reason).

Or possibly a anti-virus program.
You could try disabling any AV you have running, delete the ace folder and re-extract.

Also try right clicking on any of the dlls (like `ace_advanced_ballistics`) and going to properties.
Look for any kind of warning like "This program was downloaded from the internet and may be unsafe".
I think windows can block dlls sometimes as well.

**Comment by majinisinpu** [comment]
Windows block no dll.
I try to disable AVG-Free.

**Comment by majinisinpu** [comment]
I disabled AVG-Free and tried to play a mission.
But the issue has not solved.

**Comment by Dystopian** [comment]
- try only ACE and CBA mods
- try move mods from Documents to folder without non-English characters (you have now `C:\Users\マジニ\Documents\Arma 3\\@cba_a3`), e.g. to `C:\A3`

And it looks strange you have no `@ace3` in `-mods=` parameter.

**Comment by kripto202** [comment]
try turning off windows firewall, it could cause some problems

**Comment by majinisinpu** [comment]
I disabled windows firewall but this idea wasn't successful.

**Comment by majinisinpu** [comment]
I can't find what I do.

**Comment by majinisinpu** [comment]
New information: I reduced a lot of mods and tried only ADR-97,ACE,ACEX,CBA,RH M4,RH ACC,RH pistols,and RHS mods.
But this issue has not solved.
[Arma3_x64_2017-04-16_19-03-41.zip](https://github.com/acemod/ACE3/files/924198/Arma3_x64_2017-04-16_19-03-41.zip)

**Comment by aliasalpha** [comment]
My unit was troubleshooting this DLL loading issue for most of today and I found out that deleting the entire user profile and making a new one fixed it for us, it might be worth trying that. Note, we had to delete the .arma3profile files in the documents\arma3 folder (just those files, no need to delete missions folders or anything else), just making a new profile in the game while you already had one loaded and switching to the new one wasn't sufficient, the files had to be deleted outside the game to generate a fresh one). My suspicion is that this is nothing to do with ace and everything to do with BIS making a DLC which added new control entries since that's resulted in slightly corrupted profiles in the past, I've not seen this type of error before today but everyone in my group who had the problems and who deleted & recreated their profile said it was fixed so it seems a consistent solution.

Just to get it back on the topic of ACE; ACE is cool, thanks for all the effort folks.
